src/open_inwoner/pdc/views.py

```diff
@@ -60,7 +60,7 @@ def page_title(self):
     def get_context_data(self, **kwargs):
         config = SiteConfiguration.get_solo()
 
-        limit = 3 if self.request.user.is_authenticated else 4
+        limit = 4
```

Comment:
We can remove this variable now as the limit is not dynamic any more. Let's use the number directly in each queryset.

Reply:
In future I may have to make a difference between the mobile limit and the desktop limit, which hopefully can be done in the templates somewhere (not sure yet), so I guess this can be cleaned up.

Reply:
@vaszig  I now see the 'limit' is used 5 times in this file; is it really a good idea to remove the 'limit' variable? I would think it is cleaner to keep it and reuse it instead of the value? (in javascript I would keep the variable, not sure if this is also Python convention).

Reply:
I would use a variable if I wanted first to evaluate something and then reuse it (Not in Python, in every language). Here we don;t need to evaluate something, it's just an integer. But this is my knowledge, @Bartvaderkin can you tell us your opinion please?

Reply:
@vaszig @jiromaykin I would keep the variable, so we can see all the usages are linked to the same value. If we'd put the plain number everywhere you get a magic number situation where it isn't clear which numbers are related or just happen to have the same value.

Reply:
@Bartvaderkin @vaszig Hoorrah! in Javascript I would use a '**const**' for numbers like these - so I declare this PR as finished and ready for review again :-)